services/common/src/redux/slices/spatialDataSlice.ts

```diff
@@ -99,19 +102,39 @@ const spatialSlice = createAppSlice({
         }).get(url);
 
         thunkAPI.dispatch(hideLoading());
-        return { mapData: response.data, bundle_id };
+        return { mapData: response.data };
       },
       {
         fulfilled: (state, action) => {
           state.geoJsonData = action.payload.mapData;
-          state.bundle_id = action.payload.bundle_id.toString();
         },
         rejected: (_state, action) => {
           rejectHandler(action);
         },
       }
     ),
-    createSpatialBundle: create.asyncThunk(
+    fetchSpatialBundle: create.asyncThunk(
+      async (mine_document_bundle_id: string, thunkAPI) => {
+        thunkAPI.dispatch(showLoading());
+        const headers = createRequestHeader();
+        const url = `${ENVIRONMENT.apiUrl}/mines/document-bundle/${mine_document_bundle_id}`;
```

Comment:
Could we put this into `@mds/common/constants/API` together with teh rest of our endpoints?
And on that note: Any way to update the mock server to also use those constants?